pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -5412,7 +5420,7 @@
         "4.4.0",
         "4.5.0"
       ],
-      "BaselineVersion": "4.5.0",
+      "BaselineVersion": "5.0.0",
```

Comment:
@ericstj is it expected this would've been 4.5.0?

Reply:
Yep, please undo